src/styles/SearchBar.module.css

```diff
@@ -0,0 +1,21 @@
+.dataResults {
```

Comment:
I believe case in CSS is `kebab-case` and not `camelCase` (as in JavaScript) 😅 

So `dataResults` should be `data-results`, and `dataItem` should be `data-item`.